src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts

```diff
@@ -137,9 +126,11 @@ export class JobResultTableComponent implements AfterViewInit, OnInit, OnDestroy
     private store: Store<AppState>,
     private router: Router,
     private dialog: MatDialog,
-  ) {}
+  ) {
+  }
```

Comment:
There are some style changes like this (changed indentation, spaces etc), it would be nice to add those in separate commits so it's easier to filter on the actual changes.

Reply:
ngOnInit is executed every time that you go to the component, so the values will change :)